vulnerabilities/importers/redhat.py

```diff
@@ -40,9 +43,9 @@ def __enter__(self):
     def updated_advisories(self):
         processed_advisories = []
         for advisory_data in self.redhat_response:
-            processed_advisories.append(to_advisory(advisory_data))
+            yield [to_advisory(advisory_data)]
```

Comment:
That was for convenience in testing db writes, reverting it.

Reply:
My point was not about the yield... but the yield of a list.

Reply:
I see,  FWIW  the definition of `batch_advisories`  is . It yields sets of advisories. The importer_runner  expects the `updated_advisories` to yield a BATCH of advisories.

```
    def batch_advisories(self, advisories: List[Advisory]) -> Set[Advisory]:
        """
        Yield batches of the passed in list of advisories.
        """
        advisories = advisories[:]  # copy the list as we are mutating it in the loop below

        while advisories:
            b, advisories = advisories[: self.batch_size], advisories[self.batch_size:]
            yield set(b)
```

I don't see any significant memory-use reduction in this case since the data is not too much.

Reply:
ok, let's keep it as is then, but batching is an optimization that would likely be best done later, as it may not help with clarity ;)